src/olympia/bandwagon/views.py

```diff
@@ -110,6 +111,12 @@ class CollectionAddonViewSet(ModelViewSet):
                               'added': 'created'}
     ordering = ('-addon__weekly_downloads',)
 
+    # This endpoint can be quite slow so it's cached for one hour for all
+    # anonymous users.
+    @method_decorator(cache_page_if_anonymous(60 * 60 * 1))
+    def list(self, *args, **kwargs):
+        return super().list(*args, **kwargs)
```

Comment:
CollectionViewSet.detail with `?with_addons` returns a very similar response to this if we want to cover all the bases.

Reply:
I started looking at that, mumbling about the risk of extending the cache to collection metadata, and then realized... `with_addons` doesn't seem to be used anywhere ? So I don't think this is worth doing.

Reply:
https://github.com/mozilla/addons-server/issues/6548 it was requested for addons-frontend to use.  If it's not used we could arguably drop it... (though I think it's a quite useful feature really)